src/15utility.js

```diff
@@ -541,7 +562,15 @@ var removeFile = utils.removeFile = function(path,cb) {
                 cb && cb(); // jshint ignore:line
             });
         });
-      	//*/
+	} else if(utils.isReactNative) {
+        /* If ReactNative */

```

Comment:
As this is also stated in the `If` just above, I recommend ` /* If ReactNative */`gets removed